Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM maven:3-eclipse-temurin-11-alpine AS builder
```

Comment:
I am fine with this choice, although I can recommend [SLE BCI](https://registry.suse.com/) images as well, as they are maintained by the company.

Not blocking for this PR anyway.

Reply:
Done, thanks for the suggestion!
The resulting images are slightly larger than when using alpine as a base (335MB vs 307MB), but that looks reasonable.